docs/source/Installation.rst

```diff
@@ -302,50 +275,117 @@ From the conda environment you used to install CaImAn:
-the defaults channel; we recommend only using conda-forge. To ensure
-you’re not mixing channels, perform the install (inside your
-environment) as follows:
+For **Windows** run the same commands, replacing the word ```export``` with the word ```set```.
+
+The commands should be run *every time* before launching python. It is
+recommended that you save these values inside your environment so you
+don’t have to repeat this process every time. You can do this by
+following the instructions
+`here <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#saving-environment-variables>`__.
+
+**If you installed using the conda-forge package, this is done automatically for you.**
+
+
+Section 4B: Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+32G RAM is required for a good experience, and depending on datasets, more may be necessary. Caiman is optimized for use by 
+multiple CPUs, so workstations or clusters with multiple CPU cores are ideal (8+ logical cores). GPU computation is not used 
+heavily by Caiman (though see Section 4D). 
+
+Right now, Caiman works and is supported on the following platforms:
+
+- Linux on 64-bit x86 CPUs
+- MacOS on 64-bit x86 CPUs
+- Windows on 64-bit x86 CPUs
+
+ARM-based versions of Apple hardware work (if on a 16G model), but currently happen under x86 emulation and we cannot 
+support them as well.  Support for Linux on ARM (e.g. AWS Graviton) is not available (but it may work with the port of conda, 
+if you compile Caiman yourself - we do not have binary packages and this is untested). If you care about this, please let us know.
+
+
+Section 4C: Installing additional packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Caiman installs through the conda-forge conda channel. Some packages are available on multiple conda channels, and in this 
+case it is important that you use the conda-forge channel if possible. To do this, when installing new packages 
+inside your environment, use the following command:
 
 ::
 
    mamba install -c conda-forge --override-channels NEW_PACKAGE_NAME
 
-You will notice that any packages installed this way will mention, in
-their listing, that they’re from conda-forge, with none of them having a
-blank origin. If you fail to do this, differences between how packages
-are built in conda-forge versus the default conda channels may mean that
-some packages (e.g. OpenCV) stop working despite showing as installed.
+You will notice that any packages installed this way will mention, in their listing, 
+that they are from conda-forge, with none of them having a blank origin. If you don't do this, 
+differences between how packages are built in different channels could lead to some packages failing to work
+(e.g., OpenCV). 
+
+Section 4D: Other topics
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- `Running Caiman on a cluster <./Cluster.md>`_ 
+- `Setting up Caiman to use your GPUs <./README-GPU.md>`_
+- `Install quirks on some older Linux distributions <./README-Distros.md>`_ 
```

Comment:
It would be better to remove README-Distros.md at this point (and remove references to it).